<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1126</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionGo_back"/>
    <addaction name="actionGo_forward"/>
    <addaction name="separator"/>
    <addaction name="actionregister"/>
    <addaction name="actionLogin"/>
    <addaction name="separator"/>
    <addaction name="actionlogout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionShare"/>
    <addaction name="actionPrint"/>
    <addaction name="actionDownload"/>
    <addaction name="actionRename_2"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="actionRename"/>
    <addaction name="actionCreate_a_copy"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuFormato">
    <property name="title">
     <string>Formato</string>
    </property>
    <widget class="QMenu" name="menuTesto">
     <property name="title">
      <string>Text</string>
     </property>
     <addaction name="actionBold"/>
     <addaction name="actionItalic"/>
     <addaction name="actionULined"/>
    </widget>
    <widget class="QMenu" name="menuText_Alignment">
     <property name="title">
      <string>Text Alignment</string>
     </property>
     <addaction name="actionLeft"/>
     <addaction name="actionRight"/>
     <addaction name="actionCenter"/>
    </widget>
    <addaction name="menuTesto"/>
    <addaction name="actionFont"/>
    <addaction name="menuText_Alignment"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuModifica">
    <property name="title">
     <string>Modify</string>
    </property>
    <addaction name="actionPrevious_ctrl_z"/>
    <addaction name="actionRepeat_ctrl_y"/>
    <addaction name="separator"/>
    <addaction name="actionCut_ctrl_x"/>
    <addaction name="actionCopy_ctrl_c"/>
    <addaction name="actionPaste_ctrl_v"/>
    <addaction name="actionSelect_all_ctrl_a"/>
    <addaction name="separator"/>
    <addaction name="actionComment"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionConnected_users"/>
    <addaction name="actionComments"/>
   </widget>
   <addaction name="menumenu"/>
   <addaction name="menuFile"/>
   <addaction name="menuModifica"/>
   <addaction name="menuFormato"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionregister">
   <property name="text">
    <string>register</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>login</string>
   </property>
  </action>
  <action name="actionlogout">
   <property name="text">
    <string>logout</string>
   </property>
  </action>
  <action name="actionShare">
   <property name="text">
    <string>Share</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionCreate_a_copy">
   <property name="text">
    <string>Create a copy</string>
   </property>
  </action>
  <action name="actionGo_back">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Go back</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionULined">
   <property name="text">
    <string>ULined</string>
   </property>
  </action>
  <action name="actionPrevious_ctrl_z">
   <property name="text">
    <string>Previous </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRepeat_ctrl_y">
   <property name="text">
    <string>Repeat </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut_ctrl_x">
   <property name="text">
    <string>Cut </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy_ctrl_c">
   <property name="text">
    <string>Copy </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste_ctrl_v">
   <property name="text">
    <string>Paste </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionGo_forward">
   <property name="text">
    <string>Go forward</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="text">
    <string>Download</string>
   </property>
  </action>
  <action name="actionRename_2">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionSelect_all_ctrl_a">
   <property name="text">
    <string>Select all </string>
   </property>
  </action>
  <action name="actionConnected_users">
   <property name="text">
    <string>Connected users</string>
   </property>
  </action>
  <action name="actionComments">
   <property name="text">
    <string>Comments</string>
   </property>
  </action>
  <action name="actionComment">
   <property name="text">
    <string>Comment</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="text">
    <string>Center</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
